forem/forem
# Update Gitpod's setup

## What type of PR is this? (check all applicable)

## Description
Update Gitpod to 'use' our docker-compose instead of their version of bare-metal. What this mean is instead of custom dockerfile `.gitpod.dockerfile`, we will use their default provide base image `workspace` where we will run our docker command in. 

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
You can test this by opening this link https://gitpod.io/#https://github.com/forem/forem/pull/20026

Please give it about a minute to load the prebuild cache properly and you should see server up and running.
## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
